1.initial-z-sweep/README.md

```diff
@@ -98,45 +117,70 @@ Training of models with optimal hyperparameters are shown in **figure 3**.
 
 ### ADAGE
 
-ADAGE models had variable performance across models and failed to converge with high levels of sparsity (**Figure 4**).
+### Untied Weights
+
+With untied weights, ADAGE models had variable performance across models and failed to converge with high levels of sparsity (**Figure 4**).
 High levels of sparsity fail _worse_ with increasing dimensionality.
 
-![](figures/z_param_adage_tied_weights/z_parameter_adage_tiedweights.png?raw=true)
+![](figures/z_param_adage/z_parameter_adage.png?raw=true)
 
-**Figure 4.** The loss of validation sets at the end of training for all 1,080 ADAGE models.
+**Figure 4.** The loss of validation sets at the end of training for all 1,080 untied weight ADAGE models.
 
 After removing `sparsity = 0.001`, we see a clearer picture (**Figure 5**).
 
-![](figures/z_param_adage_tied_weights/z_parameter_adage_remove_sparsity_tiedweights.png?raw=true)
+![](figures/z_param_adage/z_parameter_adage_remove_sparsity.png?raw=true)
 
-**Figure 5.** The loss of validation sets at the end of training for 720 ADAGE models.
+**Figure 5.** The loss of validation sets at the end of training for 720 untied weight ADAGE models.
 
 A similar pattern appears where lower dimensionality benefits from increased sparsity.
 ADAGE models are also generally stable, particularly at high dimensions.
 
+It appears that `learning rate` is globally optimal at 0.0005; epochs at 100; batch size at 50; sparsity at 0; with decreasing noise for larger z dimensions.
+
+![](figures/z_param_adage/z_parameter_adage_bes.png?raw=true)
```

Comment:
This figure does not appear to be showing up here: https://github.com/gwaygenomics/interpret-compression/blob/c6bc2e39f80ecdc62078d4c92f77e2b8b86d583a/1.initial-z-sweep/README.md

Reply:
ah, yeah, typo. My bad